atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -0,0 +1,207 @@
+"""
+Copyright (c) 2015 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+from email.mime.text import MIMEText
+import smtplib
+import socket
+try:
+    from urlparse import urljoin
+except ImportError:
+    from urllib.parse import urljoin
+
+from dockerfile_parse import DockerfileParser
+import requests
+
+from atomic_reactor.plugin import ExitPlugin, PluginFailedException
+from atomic_reactor.plugins.pre_check_and_set_rebuild import is_rebuild
+from atomic_reactor.source import GitSource
+
+
+class SendMailPlugin(ExitPlugin):
+    key = "sendmail"
+    allowed_states = set(['manual_success', 'manual_fail', 'auto_success', 'auto_fail',
+                          'auto_canceled'])
```

Comment:
It would be nice to have symbolic constants for these strings, like

```
MANUAL_SUCCESS = 'manual_success'
...
allowed_states = set([self.MANUAL_SUCCESS, ...])
```

That way typos will be caught more quickly.